### PR TITLE
chore: re-add quality dashboard

### PR DESCRIPTION
**Pull request by psturc**
https://issues.redhat.com/browse/KONFLUX-2244

Based on this PR: https://github.com/redhat-appstudio/infra-deployments/pull/2676

Deployment of QD can be validated on a dev cluster by following instructions in `docs/misc/quality-dashboard.md`

### CODE REVIEW

**Comment by psturc** [comment]
Hi @gbenhaim could you please give it a look?

File: components/quality-dashboard/base/rbac/quality-dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: quality-dashboard-maintainers
+  namespace: quality-dashboard
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: konflux-qe-admins
```
Comment (gbenhaim) [inline components/quality-dashboard/base/rbac/quality-dashboard.yaml:25]: why this group need permissions to access all the secrets in the cluster?

**Review by gbenhaim — CHANGES_REQUESTED** [review]

File: components/quality-dashboard/base/rbac/quality-dashboard.yaml (hunk 1 above)
Comment (gbenhaim) [reply components/quality-dashboard/base/rbac/quality-dashboard.yaml:25]: I agree, the cluster rule can be changed to a rule.

Comment (psturc) [reply components/quality-dashboard/base/rbac/quality-dashboard.yaml:25]: thanks, addressed in https://github.com/redhat-appstudio/infra-deployments/pull/3370/commits/490a793af6ab59c2c2d77dafdcbc056381d24aa7

**Review by hugares — APPROVED** [review]
/lgtm

**Pull request merged**